app/templates/buyers/cancel_brief.html

```diff
@@ -34,7 +34,7 @@
           {%
```

Comment:
If you visit this route from the cancel_or_award flow you will end up at the url ending `cancel-award`.

If you don't pick an answer, your form will be invalid when you submit it. It will be submitted to the `url_for('.cancel_brief')` which will then re render the at the url ending `cancel` with different title and hint text to the page you started at?

Is this correct or have I made a mistake/missed something?

Reply:
Fixed here:
https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/31/files#diff-98d4a0916d4357714af043c5b6347367R32

Uses `request.endpoint` like the view

Reply:
Also tests:
https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/31/files#diff-9a0721b5c761c2fccb366192a3212493R3634
https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/31/files#diff-9a0721b5c761c2fccb366192a3212493R3676